package.json

```diff
@@ -30,48 +30,48 @@
   "dependencies": {
     "@slack/logger": "^2.0.0",
     "@slack/rtm-api": "^5.0.5",
-    "@slack/web-api": "^5.12.0",
-    "Slackdown": "Half-Shot/slackdown#efd8934a3d9c3bf0064c0b217c5cf6b62ee697e4",
-    "axios": "^0.20.0",
+    "@slack/web-api": "^5.15.0",
+    "Slackdown": "github:Half-Shot/slackdown#efd8934a3d9c3bf0064c0b217c5cf6b62ee697e4",
```

Comment:
Why off mainline? Do we have an issue to track this?

Reply:
I ended up creating https://github.com/matrix-org/matrix-appservice-slack/issues/561. Thanks for keeping me honest :)